tests/index.js

```diff
@@ -815,6 +815,14 @@ describe('Thorn', () => {
             });
 
             it('should use internal usernames for server interactions', function*() {
+                let uuidv4Format = [
+                    '[a-fA-F0-9]{8}',
+                    '[a-fA-F0-9]{4}',
+                    '4{1}[a-fA-F0-9]{3}',
+                    '[89abAB]{1}[a-fA-F0-9]{3}',
+                    '[a-fA-F0-9]{12}'
```

Comment:
Please place a trailing comma at the end of this line.

For future reference, you can run `gulp lint` before submitting a PR to make sure your code is style-guide compliant.